python/core/auto_generated/qgsdistancearea.sip.in

```diff
@@ -224,6 +224,31 @@ Measures the distance between two points.
 
 :return: distance between points. The units for the returned distance can be retrieved by calling lengthUnits().
 
+.. seealso:: :py:func:`lengthUnits`
+%End
+
+    double measureLine3D( const QVector<QgsPoint> &points ) const;
+%Docstring
+Measures the length of a line with multiple segments in 3D.
+Support cartesian calculation only.
```

Comment:
I don't think this belongs in QgsDistanceArea if it has no handling of ellipsoidal/geodesic calculations. Rather it belongs in the geometry classes themselves.

See https://github.com/qgis/qgis4.0_api/issues/90 - this class should actually be named "QgsGeodesicUtils".

Reply:
Moved.